<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" 
   "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">

<head>
<title>SAAP analysis</title>

<script type='text/javascript' src='/js/getElementsByClassName.js'>
<!-- http://robertnyman.com -->
</script>
<script type="text/javascript" src="/js/overlib/overlib.js">
<!-- overLIB (c) Erik Bosrup -->
</script>

<script type='text/javascript'>
<!--


function toggle_text(obj) 
{
   var text = document.getElementById(obj);
   if ( text.style.display == 'inline' ) { text.style.display = 'none'; }
   else { text.style.display = 'inline'; }
}
function toggle_text_arrow(obj1, obj2) 
{
   var text = document.getElementById(obj1);
   var arrow = document.getElementById(obj2);
   if (( text.style.display == 'inline' ) || ( text.style.display == 'block' ))
   { 
      text.style.display = 'none'; 
      arrow.className = 'fliparrow fa fa-toggle-down';
   }
   else 
   { 
      text.style.display = 'inline'; 
      arrow.className = 'fliparrow fa fa-toggle-up';
   }
}
function expand_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'block';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-up';
   }
}

function collapse_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'none';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-down';
   }
}

function fixIE()
{
//    document.getElementsByClassName("submit").disabled = false;

    if (document.getElementsByClassName == undefined) 
    {
        document.getElementsByClassName = function(className)
        {
            var hasClassName = new RegExp("(?:^|\s)" + className + "(?:0\s)");
            var allElements = document.getElementsByTagName("*");
            var results = [];
            
            var element;
            for (var i = 0; (element = allElements[i]) != null; i++) 
            {
                var elementClass = element.className;
                if (elementClass && elementClass.indexOf(className) != -1 && 
                    hasClassName.test(elementClass))
                {
                    results.push(element);
                }
            }
            
            return results;
        }
    }
}

-->
</script>

<!-- Ajax for Predict button -->
<script type='text/javascript' src='/saap/dap//predictAjax.js'></script>

<link rel='stylesheet' href='/css/font-awesome/css/font-awesome.min.css' />

<style type='text/css'>
<!--
p, td, th, li {  font-family: Helvetica,Arial; }

h1 
{  text-align: center;
   font: 24pt Helvetica,Arial
}

h2 { font: 18pt Helvetica,Arial;      }
h3 { font: bold 14pt Helvetica,Arial; }
h4 { font: 12pt Helvetica,Arial;      }
a  { outline: none;                   }

a span, a:active, a:visited, a:hover 
{  border: none; 
   color: #000000;
   text-decoration: none;
}

.mutation 
{  background: #666666;
   margin: 0em 0em 1em 0em;
   padding: 2pt;
}

.mutation td 
{  color: #FFFFFF;
   font: 12pt Helvetica, Arial;
}

.summary .ok      {background: #cccccc;}
.summary .bad     {background: #FF0000;}
.summary .unknown {background: #AAAAAA;}

.summary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: center;
}

.summary td, .summary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 10pt Helvetica,Arial;
}

.summary th { font: 12pt Helvetica,Arial; }

.summary th a:link, .summary th a:visited, .summary th a:hover, .summary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.mutantsummary .ok      {background: #00FF00;}
.mutantsummary .bad     {background: #FF0000;}
.mutantsummary .unknown {background: #AAAAAA;}

.mutantsummary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: left;
}

.mutantsummary td, .mutantsummary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 12pt Helvetica,Arial;
}

.mutantsummary th a:link, .mutantsummary th a:visited, .mutantsummary th a:hover, .mutantsummary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.footnote 
{  font: italic 8pt Helvetica,Arial;
}

.button 
{  font: bold 12pt Helvetica,Arial;
   background: #7A96BE;
   margin: 4pt 0pt;
   padding: 1pt 4pt 4pt 4pt;
   border: outset 2pt #000000;
}

.button a:link, .button a:visited, .button a:hover, .button a:active  
{  font: bold 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.analysis 
{  border: 1pt solid #000000;
   margin: 1em 0 0 0;
   padding: 0 0 0 0;
   background: #CCCCCC;
}

.analysis h3 
{  margin: 0 0 0 0;
   padding: 2pt;
   background: #666666;
   color: #FFFFFF;
}

.analysis h4 
{  margin: 0 0 0 0;
   padding: 0 2pt 2pt 2pt;
   background: #aaaaaa;
   color: #000000;
   border: solid 1pt #777777;
}

.analysis .left  { width: 49%; }
.analysis .right { width: 49%; }

.analysis .bad 
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #ff0000;
}

.analysis .ok  
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #eeeeee;
}

.analysis .unknown  
{  margin: 0;
   padding: 2pt;
   background: #AAAAAA;
   border-bottom: solid 1pt #7A96BE;
   border-left: solid 1pt #7A96BE;
   border-right: solid 1pt #7A96BE;
}

.analysis p 
{  margin: 0;
   padding: 0;
}

.analysis table { width: 100%; }
.analysis td    {vertical-align: top; }

.resultsFrame
{  background: #FFFFFF;
   border: solid 1px #666666;
   padding: 1em;
}

-->
</style>
</head>
<body>
<!-- <body onload='fixIE();'> -->
<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>

<h1>SAAP Analysis</h1>

<!-- END OF COMMON HEADER -->


<!-- INFO ON THE MUTATION -->
<div class='mutation'>
<table>
<tr><td>UniProt Entry:</td><td>P12883</td></tr>
<tr><td>Mutation:</td><td>K 847-&gt;E</td></tr>
</table>
</div>
<!-- END OF INFO ON THE MUTATION -->

<!-- BEGIN SUMMARY -->
<h2>Summary</h2>

<div class='summary'>
<table>
    <tr>
        <th>PDB</th>
        <th>chain</th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation disrupts hydrogen bonding.');" 
        onmouseout="return nd();">HBonds</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was involved in a specific HBond or packing interaction with another protein chain or ligand.');" 
        onmouseout="return nd();">Binding</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a charged residue into the protein core.');" 
        onmouseout="return nd();">BuriedCharge</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a void in the protein core.');" 
        onmouseout="return nd();">Voids</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is annotated as a feature in UniProtKB/SwissProt.');" 
        onmouseout="return nd();">SProtFT</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a hydrophobic amino acid on the surface.');" 
        onmouseout="return nd();">SurfacePhobic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was in an interface.');" 
        onmouseout="return nd();">Interface</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a glycine which was adopting backbone torsions not allowed for other amino acids.');" 
        onmouseout="return nd();">Glycine</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces an amino acid which clashes with its surroundings.');" 
        onmouseout="return nd();">Clash</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a cis-proline.');" 
        onmouseout="return nd();">CisPro</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a proline when native residue was in a conformation not accessible to proline.');" 
        onmouseout="return nd();">Proline</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a hydrophilic residue into the protein core.');" 
        onmouseout="return nd();">CorePhilic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is significantly conserved.');" 
        onmouseout="return nd();">Impact</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was a cysteine involved in a disulphide bond.');" 
        onmouseout="return nd();">SSGeom</a>
        </th>
    </tr>
        <tr><td align='left'><a href='#P2fxoB'>2fxo</a> </td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P2fxoC'>2fxo</a> </td>
            <td>C</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P3dtpA'>3dtp</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P3dtpB'>3dtp</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P4p7hA'>4p7h</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P4p7hB'>4p7h</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P4pa0A'>4pa0</a> </td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P4pa0B'>4pa0</a> </td>
            <td>B</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5tbyA'>5tby</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5tbyB'>5tby</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
</table>
<p class='footnote'>Hover over the column titles for an explanation.</p>
</div>

<!-- END SUMMARY -->
<!-- BEGIN STRUCTURAL ANALYSIS HEADER -->
<h2>Structural analysis</h2>

<div style='text-align: center;'>
<button onclick='expand_all();'>Expand all</button>
&nbsp;&nbsp; 
<button onclick='collapse_all();'>Collapse all</button>
</div>

<!-- END STRUCTURAL ANALYSIS HEADER -->

<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P2fxoB'>2fxo Residue B847 | Structure type: crystal | Resolution: 2.50A | R-factor: 27.70%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P2fxoBHBonds','P2fxoBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P2fxoBBinding','P2fxoBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P2fxoBBuriedCharge','P2fxoBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 62.642%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P2fxoBVoids','P2fxoBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 9.74</b>.<br /> 
The largest void in the native was 9.74.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P2fxoBSProtFT','P2fxoBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P2fxoBSurfacePhobic','P2fxoBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 62.642%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P2fxoBInterface','P2fxoBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 62.642% 
    while the individual chain had an accessibility of 62.642%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P2fxoBGlycine','P2fxoBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBGlycine' class='collapsable' style='display: none'>
           The native residue was a LYS not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P2fxoBClash','P2fxoBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -5.47&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P2fxoBCisPro','P2fxoBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P2fxoBProline','P2fxoBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P2fxoBCorePhilic','P2fxoBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 62.642%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P2fxoBImpact','P2fxoBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P2fxoBSSGeom','P2fxoBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P2fxoBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P2fxoC'>2fxo Residue C847 | Structure type: crystal | Resolution: 2.50A | R-factor: 27.70%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P2fxoCHBonds','P2fxoCHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P2fxoCBinding','P2fxoCBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P2fxoCBuriedCharge','P2fxoCBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 72.132%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P2fxoCVoids','P2fxoCVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 10.10</b>.<br /> 
The largest void in the native was 10.10.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P2fxoCSProtFT','P2fxoCSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P2fxoCSurfacePhobic','P2fxoCSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 72.132%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P2fxoCInterface','P2fxoCInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 72.132% 
    while the individual chain had an accessibility of 72.132%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P2fxoCGlycine','P2fxoCGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCGlycine' class='collapsable' style='display: none'>
           The native residue was a LYS not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P2fxoCClash','P2fxoCClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCClash' class='collapsable' style='display: none'>
           <b>The clash energy was -1.10&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P2fxoCCisPro','P2fxoCCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P2fxoCProline','P2fxoCProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P2fxoCCorePhilic','P2fxoCCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 72.132%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P2fxoCImpact','P2fxoCImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P2fxoCImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P2fxoCSSGeom','P2fxoCSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P2fxoCSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P2fxoCSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P3dtpA'>3dtp Residue A858 | Structure type: ElectronMicroscopy | Resolution: 20.00A <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P3dtpAHBonds','P3dtpAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P3dtpABinding','P3dtpABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P3dtpABuriedCharge','P3dtpABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 69.170%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P3dtpAVoids','P3dtpAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 770.25</b>.<br /> 
The largest void in the native was 770.25.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P3dtpASProtFT','P3dtpASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P3dtpASurfacePhobic','P3dtpASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 69.170%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P3dtpAInterface','P3dtpAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 69.170% 
    while the individual chain had an accessibility of 69.170%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P3dtpAGlycine','P3dtpAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpAGlycine' class='collapsable' style='display: none'>
           The native residue was a LYS not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P3dtpAClash','P3dtpAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -4.61&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P3dtpACisPro','P3dtpACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P3dtpAProline','P3dtpAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpAProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P3dtpACorePhilic','P3dtpACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 69.170%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P3dtpAImpact','P3dtpAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 50%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 86% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P3dtpASSGeom','P3dtpASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P3dtpASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P3dtpB'>3dtp Residue B858 | Structure type: ElectronMicroscopy | Resolution: 20.00A <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P3dtpBHBonds','P3dtpBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P3dtpBBinding','P3dtpBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P3dtpBBuriedCharge','P3dtpBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 70.884%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P3dtpBVoids','P3dtpBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 525.38</b>.<br /> 
The largest void in the native was 525.38.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P3dtpBSProtFT','P3dtpBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P3dtpBSurfacePhobic','P3dtpBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 70.884%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P3dtpBInterface','P3dtpBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The residue was involved in an interface</p>
        <p id='P3dtpBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 59.930% 
    while the individual chain had an accessibility of 70.884%, <b>a
    difference of 10.954%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P3dtpBGlycine','P3dtpBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBGlycine' class='collapsable' style='display: none'>
           The native residue was a LYS not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P3dtpBClash','P3dtpBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -5.31&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P3dtpBCisPro','P3dtpBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P3dtpBProline','P3dtpBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P3dtpBCorePhilic','P3dtpBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 70.884%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P3dtpBImpact','P3dtpBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P3dtpBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 50%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 86% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P3dtpBSSGeom','P3dtpBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P3dtpBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P3dtpBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P4p7hA'>4p7h Residue A847 | Structure type: crystal | Resolution: 3.20A | R-factor: 22.90% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P4p7hAHBonds','P4p7hAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P4p7hABinding','P4p7hABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P4p7hABuriedCharge','P4p7hABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation resulted in introducing or removing a buried charge.</p>
        <p id='P4p7hABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 0.230%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P4p7hAVoids','P4p7hAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 822.16</b>.<br /> 
The largest void in the native was 709.62.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P4p7hASProtFT','P4p7hASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P4p7hASurfacePhobic','P4p7hASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.53<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 0.230%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P4p7hAInterface','P4p7hAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 0.230% 
    while the individual chain had an accessibility of 0.230%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P4p7hAGlycine','P4p7hAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hAGlycine' class='collapsable' style='display: none'>
           The native residue was a LEU not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P4p7hAClash','P4p7hAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -14.44&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P4p7hACisPro','P4p7hACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P4p7hAProline','P4p7hAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hAProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P4p7hACorePhilic','P4p7hACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation introduces a hydrophilic residue into the core of the protein.</p>
        <p id='P4p7hACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.53<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 0.230%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P4p7hAImpact','P4p7hAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P4p7hASSGeom','P4p7hASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P4p7hASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P4p7hB'>4p7h Residue B847 | Structure type: crystal | Resolution: 3.20A | R-factor: 22.90% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P4p7hBHBonds','P4p7hBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P4p7hBBinding','P4p7hBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P4p7hBBuriedCharge','P4p7hBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation resulted in introducing or removing a buried charge.</p>
        <p id='P4p7hBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 0.156%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P4p7hBVoids','P4p7hBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 708.97</b>.<br /> 
The largest void in the native was 613.14.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P4p7hBSProtFT','P4p7hBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P4p7hBSurfacePhobic','P4p7hBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.53<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 0.156%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P4p7hBInterface','P4p7hBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 0.156% 
    while the individual chain had an accessibility of 0.156%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P4p7hBGlycine','P4p7hBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBGlycine' class='collapsable' style='display: none'>
           The native residue was a LEU not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P4p7hBClash','P4p7hBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -13.36&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P4p7hBCisPro','P4p7hBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P4p7hBProline','P4p7hBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P4p7hBCorePhilic','P4p7hBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation introduces a hydrophilic residue into the core of the protein.</p>
        <p id='P4p7hBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.53<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 0.156%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P4p7hBImpact','P4p7hBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4p7hBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P4p7hBSSGeom','P4p7hBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4p7hBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P4p7hBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P4pa0A'>4pa0 Residue A847 | Structure type: crystal | Resolution: 2.25A | R-factor: 20.30%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P4pa0AHBonds','P4pa0AHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0AHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0AHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P4pa0ABinding','P4pa0ABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0ABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0ABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P4pa0ABuriedCharge','P4pa0ABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0ABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation resulted in introducing or removing a buried charge.</p>
        <p id='P4pa0ABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 0.125%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P4pa0AVoids','P4pa0AVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0AVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0AVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 402.27</b>.<br /> 
The largest void in the native was 402.27.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P4pa0ASProtFT','P4pa0ASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0ASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0ASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P4pa0ASurfacePhobic','P4pa0ASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0ASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0ASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.53<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 0.125%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P4pa0AInterface','P4pa0AInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0AInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0AInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 0.125% 
    while the individual chain had an accessibility of 0.125%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P4pa0AGlycine','P4pa0AGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0AGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0AGlycine' class='collapsable' style='display: none'>
           The native residue was a LEU not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P4pa0AClash','P4pa0AClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0AClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0AClash' class='collapsable' style='display: none'>
           <b>The clash energy was -14.33&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P4pa0ACisPro','P4pa0ACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0ACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0ACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P4pa0AProline','P4pa0AProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0AProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0AProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P4pa0ACorePhilic','P4pa0ACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0ACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation introduces a hydrophilic residue into the core of the protein.</p>
        <p id='P4pa0ACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.53<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 0.125%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P4pa0AImpact','P4pa0AImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0AImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0AImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P4pa0ASSGeom','P4pa0ASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0ASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P4pa0ASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P4pa0B'>4pa0 Residue B861 | Structure type: crystal | Resolution: 2.25A | R-factor: 20.30%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P4pa0BHBonds','P4pa0BHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation disrupts a hydrogen bond.</p>
        <p id='P4pa0BHBonds' class='collapsable' style='display: none'>
           The  OH  of the native residue 
was forming a hydrogen bond with  NE2 of 
residue B986. The replacement sidechain is 
unable to maintain this hydrogen bond.
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P4pa0BBinding','P4pa0BBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P4pa0BBuriedCharge','P4pa0BBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 17.265%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P4pa0BVoids','P4pa0BVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 495.32</b>.<br /> 
The largest void in the native was 495.32.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P4pa0BSProtFT','P4pa0BSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P4pa0BSurfacePhobic','P4pa0BSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.02<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 17.265%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P4pa0BInterface','P4pa0BInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 17.265% 
    while the individual chain had an accessibility of 17.265%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P4pa0BGlycine','P4pa0BGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BGlycine' class='collapsable' style='display: none'>
           The native residue was a TYR not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P4pa0BClash','P4pa0BClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BClash' class='collapsable' style='display: none'>
           <b>The clash energy was -11.83&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P4pa0BCisPro','P4pa0BCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P4pa0BProline','P4pa0BProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P4pa0BCorePhilic','P4pa0BCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.02<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 17.265%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P4pa0BImpact','P4pa0BImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P4pa0BImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P4pa0BSSGeom','P4pa0BSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P4pa0BSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P4pa0BSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5tbyA'>5tby Residue A847 | Structure type: ElectronMicroscopy | Resolution: 20.00A <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5tbyAHBonds','P5tbyAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5tbyABinding','P5tbyABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5tbyABuriedCharge','P5tbyABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 75.153%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5tbyAVoids','P5tbyAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 656.46</b>.<br /> 
The largest void in the native was 656.46.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5tbyASProtFT','P5tbyASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5tbyASurfacePhobic','P5tbyASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 75.153%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5tbyAInterface','P5tbyAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 75.153% 
    while the individual chain had an accessibility of 75.153%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5tbyAGlycine','P5tbyAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyAGlycine' class='collapsable' style='display: none'>
           The native residue was a LYS not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5tbyAClash','P5tbyAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -8.77&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5tbyACisPro','P5tbyACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5tbyAProline','P5tbyAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyAProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5tbyACorePhilic','P5tbyACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 75.153%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5tbyAImpact','P5tbyAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5tbyASSGeom','P5tbyASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5tbyASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5tbyB'>5tby Residue B847 | Structure type: ElectronMicroscopy | Resolution: 20.00A <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5tbyBHBonds','P5tbyBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5tbyBBinding','P5tbyBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5tbyBBuriedCharge','P5tbyBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: -1</b><br />
               Relative accessibility of native residue: 72.755%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5tbyBVoids','P5tbyBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 1194.99</b>.<br /> 
The largest void in the native was 1194.99.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5tbyBSProtFT','P5tbyBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5tbyBSurfacePhobic','P5tbyBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                 <b>Mutant residue hydrophobicity: -0.62</b><br />
                 Relative accessibility of native residue: 72.755%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5tbyBInterface','P5tbyBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The residue was involved in an interface</p>
        <p id='P5tbyBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 59.977% 
    while the individual chain had an accessibility of 72.755%, <b>a
    difference of 12.778%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5tbyBGlycine','P5tbyBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBGlycine' class='collapsable' style='display: none'>
           The native residue was a LYS not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5tbyBClash','P5tbyBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -8.73&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5tbyBCisPro','P5tbyBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5tbyBProline','P5tbyBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBProline' class='collapsable' style='display: none'>
           The mutant residue was a GLU not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5tbyBCorePhilic','P5tbyBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.1<br />
                  Mutant residue hydrophobicity: -0.62<br />
                  Relative accessibility of native residue: 72.755%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5tbyBImpact','P5tbyBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5tbyBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5tbyBSSGeom','P5tbyBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5tbyBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5tbyBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<p>
<a href='./printpdf.cgi'>[Download Summary PDF]</a>
<a href='./printpdf.cgi?expand=1'>[Download Expanded PDF]</a>
</p>

<!-- COMMON FOOTER -->

</body>
</html>

