<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" 
   "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">

<head>
<title>SAAP analysis</title>

<script type='text/javascript' src='/js/getElementsByClassName.js'>
<!-- http://robertnyman.com -->
</script>
<script type="text/javascript" src="/js/overlib/overlib.js">
<!-- overLIB (c) Erik Bosrup -->
</script>

<script type='text/javascript'>
<!--


function toggle_text(obj) 
{
   var text = document.getElementById(obj);
   if ( text.style.display == 'inline' ) { text.style.display = 'none'; }
   else { text.style.display = 'inline'; }
}
function toggle_text_arrow(obj1, obj2) 
{
   var text = document.getElementById(obj1);
   var arrow = document.getElementById(obj2);
   if (( text.style.display == 'inline' ) || ( text.style.display == 'block' ))
   { 
      text.style.display = 'none'; 
      arrow.className = 'fliparrow fa fa-toggle-down';
   }
   else 
   { 
      text.style.display = 'inline'; 
      arrow.className = 'fliparrow fa fa-toggle-up';
   }
}
function expand_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'block';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-up';
   }
}

function collapse_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'none';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-down';
   }
}

function fixIE()
{
//    document.getElementsByClassName("submit").disabled = false;

    if (document.getElementsByClassName == undefined) 
    {
        document.getElementsByClassName = function(className)
        {
            var hasClassName = new RegExp("(?:^|\s)" + className + "(?:0\s)");
            var allElements = document.getElementsByTagName("*");
            var results = [];
            
            var element;
            for (var i = 0; (element = allElements[i]) != null; i++) 
            {
                var elementClass = element.className;
                if (elementClass && elementClass.indexOf(className) != -1 && 
                    hasClassName.test(elementClass))
                {
                    results.push(element);
                }
            }
            
            return results;
        }
    }
}

-->
</script>

<!-- Ajax for Predict button -->
<script type='text/javascript' src='/saap/dap//predictAjax.js'></script>

<link rel='stylesheet' href='/css/font-awesome/css/font-awesome.min.css' />

<style type='text/css'>
<!--
p, td, th, li {  font-family: Helvetica,Arial; }

h1 
{  text-align: center;
   font: 24pt Helvetica,Arial
}

h2 { font: 18pt Helvetica,Arial;      }
h3 { font: bold 14pt Helvetica,Arial; }
h4 { font: 12pt Helvetica,Arial;      }
a  { outline: none;                   }

a span, a:active, a:visited, a:hover 
{  border: none; 
   color: #000000;
   text-decoration: none;
}

.mutation 
{  background: #666666;
   margin: 0em 0em 1em 0em;
   padding: 2pt;
}

.mutation td 
{  color: #FFFFFF;
   font: 12pt Helvetica, Arial;
}

.summary .ok      {background: #cccccc;}
.summary .bad     {background: #FF0000;}
.summary .unknown {background: #AAAAAA;}

.summary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: center;
}

.summary td, .summary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 10pt Helvetica,Arial;
}

.summary th { font: 12pt Helvetica,Arial; }

.summary th a:link, .summary th a:visited, .summary th a:hover, .summary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.mutantsummary .ok      {background: #00FF00;}
.mutantsummary .bad     {background: #FF0000;}
.mutantsummary .unknown {background: #AAAAAA;}

.mutantsummary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: left;
}

.mutantsummary td, .mutantsummary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 12pt Helvetica,Arial;
}

.mutantsummary th a:link, .mutantsummary th a:visited, .mutantsummary th a:hover, .mutantsummary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.footnote 
{  font: italic 8pt Helvetica,Arial;
}

.button 
{  font: bold 12pt Helvetica,Arial;
   background: #7A96BE;
   margin: 4pt 0pt;
   padding: 1pt 4pt 4pt 4pt;
   border: outset 2pt #000000;
}

.button a:link, .button a:visited, .button a:hover, .button a:active  
{  font: bold 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.analysis 
{  border: 1pt solid #000000;
   margin: 1em 0 0 0;
   padding: 0 0 0 0;
   background: #CCCCCC;
}

.analysis h3 
{  margin: 0 0 0 0;
   padding: 2pt;
   background: #666666;
   color: #FFFFFF;
}

.analysis h4 
{  margin: 0 0 0 0;
   padding: 0 2pt 2pt 2pt;
   background: #aaaaaa;
   color: #000000;
   border: solid 1pt #777777;
}

.analysis .left  { width: 49%; }
.analysis .right { width: 49%; }

.analysis .bad 
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #ff0000;
}

.analysis .ok  
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #eeeeee;
}

.analysis .unknown  
{  margin: 0;
   padding: 2pt;
   background: #AAAAAA;
   border-bottom: solid 1pt #7A96BE;
   border-left: solid 1pt #7A96BE;
   border-right: solid 1pt #7A96BE;
}

.analysis p 
{  margin: 0;
   padding: 0;
}

.analysis table { width: 100%; }
.analysis td    {vertical-align: top; }

.resultsFrame
{  background: #FFFFFF;
   border: solid 1px #666666;
   padding: 1em;
}

-->
</style>
</head>
<body>
<!-- <body onload='fixIE();'> -->
<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>

<h1>SAAP Analysis</h1>

<!-- END OF COMMON HEADER -->


<!-- INFO ON THE MUTATION -->
<div class='mutation'>
<table>
<tr><td>UniProt Entry:</td><td>P12883</td></tr>
<tr><td>Mutation:</td><td>R 1712-&gt;Q</td></tr>
</table>
</div>
<!-- END OF INFO ON THE MUTATION -->

<!-- BEGIN SUMMARY -->
<h2>Summary</h2>

<div class='summary'>
<table>
    <tr>
        <th>PDB</th>
        <th>chain</th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation disrupts hydrogen bonding.');" 
        onmouseout="return nd();">HBonds</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was involved in a specific HBond or packing interaction with another protein chain or ligand.');" 
        onmouseout="return nd();">Binding</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a charged residue into the protein core.');" 
        onmouseout="return nd();">BuriedCharge</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a void in the protein core.');" 
        onmouseout="return nd();">Voids</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is annotated as a feature in UniProtKB/SwissProt.');" 
        onmouseout="return nd();">SProtFT</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a hydrophobic amino acid on the surface.');" 
        onmouseout="return nd();">SurfacePhobic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was in an interface.');" 
        onmouseout="return nd();">Interface</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a glycine which was adopting backbone torsions not allowed for other amino acids.');" 
        onmouseout="return nd();">Glycine</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces an amino acid which clashes with its surroundings.');" 
        onmouseout="return nd();">Clash</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a cis-proline.');" 
        onmouseout="return nd();">CisPro</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a proline when native residue was in a conformation not accessible to proline.');" 
        onmouseout="return nd();">Proline</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a hydrophilic residue into the protein core.');" 
        onmouseout="return nd();">CorePhilic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is significantly conserved.');" 
        onmouseout="return nd();">Impact</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was a cysteine involved in a disulphide bond.');" 
        onmouseout="return nd();">SSGeom</a>
        </th>
    </tr>
        <tr><td align='left'><a href='#P5wlqA'>5wlq</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wlzA'>5wlz</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wlzB'>5wlz</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wlzC'>5wlz</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>C</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wlzD'>5wlz</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>D</td>
<td class='bad'>X</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
        </tr>
</table>
<p class='footnote'>Hover over the column titles for an explanation.</p>
</div>

<!-- END SUMMARY -->
<!-- BEGIN STRUCTURAL ANALYSIS HEADER -->
<h2>Structural analysis</h2>

<div style='text-align: center;'>
<button onclick='expand_all();'>Expand all</button>
&nbsp;&nbsp; 
<button onclick='collapse_all();'>Collapse all</button>
</div>

<!-- END STRUCTURAL ANALYSIS HEADER -->

<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wlqA'>5wlq Residue A1712 | Structure type: crystal | Resolution: 3.10A | R-factor: 22.30% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wlqAHBonds','P5wlqAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wlqABinding','P5wlqABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wlqABuriedCharge','P5wlqABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 0</b><br />
               Relative accessibility of native residue: 61.479%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wlqAVoids','P5wlqAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 15.46</b>.<br /> 
The largest void in the native was 15.46.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wlqASProtFT','P5wlqASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wlqASurfacePhobic','P5wlqASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.69</b><br />
                 Relative accessibility of native residue: 61.479%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wlqAInterface','P5wlqAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 61.479% 
    while the individual chain had an accessibility of 61.479%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wlqAGlycine','P5wlqAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqAGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wlqAClash','P5wlqAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -1.76&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wlqACisPro','P5wlqACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wlqAProline','P5wlqAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqAProline' class='collapsable' style='display: none'>
           The mutant residue was a GLN not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wlqACorePhilic','P5wlqACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.69<br />
                  Relative accessibility of native residue: 61.479%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wlqAImpact','P5wlqAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlqAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wlqASSGeom','P5wlqASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlqASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wlqASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wlzA'>5wlz Residue A1712 | Structure type: crystal | Resolution: 3.50A | R-factor: 21.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wlzAHBonds','P5wlzAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wlzABinding','P5wlzABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wlzABuriedCharge','P5wlzABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 0</b><br />
               Relative accessibility of native residue: 62.781%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wlzAVoids','P5wlzAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 197.34</b>.<br /> 
The largest void in the native was 197.34.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wlzASProtFT','P5wlzASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wlzASurfacePhobic','P5wlzASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.69</b><br />
                 Relative accessibility of native residue: 62.781%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wlzAInterface','P5wlzAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The residue was involved in an interface</p>
        <p id='P5wlzAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 40.797% 
    while the individual chain had an accessibility of 62.781%, <b>a
    difference of 21.984%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wlzAGlycine','P5wlzAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzAGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wlzAClash','P5wlzAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -4.15&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wlzACisPro','P5wlzACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wlzAProline','P5wlzAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzAProline' class='collapsable' style='display: none'>
           The mutant residue was a GLN not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wlzACorePhilic','P5wlzACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.69<br />
                  Relative accessibility of native residue: 62.781%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wlzAImpact','P5wlzAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wlzASSGeom','P5wlzASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wlzASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wlzB'>5wlz Residue B1712 | Structure type: crystal | Resolution: 3.50A | R-factor: 21.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wlzBHBonds','P5wlzBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wlzBBinding','P5wlzBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wlzBBuriedCharge','P5wlzBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 0</b><br />
               Relative accessibility of native residue: 63.078%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wlzBVoids','P5wlzBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 120.61</b>.<br /> 
The largest void in the native was 120.61.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wlzBSProtFT','P5wlzBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wlzBSurfacePhobic','P5wlzBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.69</b><br />
                 Relative accessibility of native residue: 63.078%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wlzBInterface','P5wlzBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The residue was involved in an interface</p>
        <p id='P5wlzBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 50.706% 
    while the individual chain had an accessibility of 63.078%, <b>a
    difference of 12.372%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wlzBGlycine','P5wlzBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wlzBClash','P5wlzBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -4.58&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wlzBCisPro','P5wlzBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wlzBProline','P5wlzBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBProline' class='collapsable' style='display: none'>
           The mutant residue was a GLN not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wlzBCorePhilic','P5wlzBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.69<br />
                  Relative accessibility of native residue: 63.078%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wlzBImpact','P5wlzBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wlzBSSGeom','P5wlzBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wlzBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wlzC'>5wlz Residue C1712 | Structure type: crystal | Resolution: 3.50A | R-factor: 21.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wlzCHBonds','P5wlzCHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wlzCBinding','P5wlzCBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wlzCBuriedCharge','P5wlzCBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 0</b><br />
               Relative accessibility of native residue: 65.853%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wlzCVoids','P5wlzCVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 245.71</b>.<br /> 
The largest void in the native was 245.71.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wlzCSProtFT','P5wlzCSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wlzCSurfacePhobic','P5wlzCSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.69</b><br />
                 Relative accessibility of native residue: 65.853%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wlzCInterface','P5wlzCInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The residue was involved in an interface</p>
        <p id='P5wlzCInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 45.655% 
    while the individual chain had an accessibility of 65.853%, <b>a
    difference of 20.198%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wlzCGlycine','P5wlzCGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wlzCClash','P5wlzCClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCClash' class='collapsable' style='display: none'>
           <b>The clash energy was -4.12&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wlzCCisPro','P5wlzCCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wlzCProline','P5wlzCProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCProline' class='collapsable' style='display: none'>
           The mutant residue was a GLN not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wlzCCorePhilic','P5wlzCCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.69<br />
                  Relative accessibility of native residue: 65.853%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wlzCImpact','P5wlzCImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzCImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wlzCSSGeom','P5wlzCSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzCSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wlzCSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wlzD'>5wlz Residue D1712 | Structure type: crystal | Resolution: 3.50A | R-factor: 21.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wlzDHBonds','P5wlzDHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation disrupts a hydrogen bond.</p>
        <p id='P5wlzDHBonds' class='collapsable' style='display: none'>
           The  NH2 of the native residue 
was forming a hydrogen bond with  ND1 of 
residue C1717. The replacement sidechain is 
unable to maintain this hydrogen bond.
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wlzDBinding','P5wlzDBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDBindingArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The native residue was involved in binding</p>
        <p id='P5wlzDBinding' class='collapsable' style='display: none'>
           A specific HBond or van der Waals interaction occurred with another protein or ligand
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wlzDBuriedCharge','P5wlzDBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 0</b><br />
               Relative accessibility of native residue: 63.903%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wlzDVoids','P5wlzDVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 149.38</b>.<br /> 
The largest void in the native was 149.38.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wlzDSProtFT','P5wlzDSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wlzDSurfacePhobic','P5wlzDSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.69</b><br />
                 Relative accessibility of native residue: 63.903%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wlzDInterface','P5wlzDInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The residue was involved in an interface</p>
        <p id='P5wlzDInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 43.075% 
    while the individual chain had an accessibility of 63.903%, <b>a
    difference of 20.828%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wlzDGlycine','P5wlzDGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wlzDClash','P5wlzDClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDClash' class='collapsable' style='display: none'>
           <b>The clash energy was -3.40&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wlzDCisPro','P5wlzDCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wlzDProline','P5wlzDProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDProline' class='collapsable' style='display: none'>
           The mutant residue was a GLN not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wlzDCorePhilic','P5wlzDCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.69<br />
                  Relative accessibility of native residue: 63.903%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wlzDImpact','P5wlzDImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDImpactArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wlzDImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 61%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wlzDSSGeom','P5wlzDSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wlzDSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wlzDSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<p>
<a href='./printpdf.cgi'>[Download Summary PDF]</a>
<a href='./printpdf.cgi?expand=1'>[Download Expanded PDF]</a>
</p>

<!-- COMMON FOOTER -->

</body>
</html>

