<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" 
   "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">

<head>
<title>SAAP analysis</title>

<script type='text/javascript' src='/js/getElementsByClassName.js'>
<!-- http://robertnyman.com -->
</script>
<script type="text/javascript" src="/js/overlib/overlib.js">
<!-- overLIB (c) Erik Bosrup -->
</script>

<script type='text/javascript'>
<!--


function toggle_text(obj) 
{
   var text = document.getElementById(obj);
   if ( text.style.display == 'inline' ) { text.style.display = 'none'; }
   else { text.style.display = 'inline'; }
}
function toggle_text_arrow(obj1, obj2) 
{
   var text = document.getElementById(obj1);
   var arrow = document.getElementById(obj2);
   if (( text.style.display == 'inline' ) || ( text.style.display == 'block' ))
   { 
      text.style.display = 'none'; 
      arrow.className = 'fliparrow fa fa-toggle-down';
   }
   else 
   { 
      text.style.display = 'inline'; 
      arrow.className = 'fliparrow fa fa-toggle-up';
   }
}
function expand_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'block';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-up';
   }
}

function collapse_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'none';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-down';
   }
}

function fixIE()
{
//    document.getElementsByClassName("submit").disabled = false;

    if (document.getElementsByClassName == undefined) 
    {
        document.getElementsByClassName = function(className)
        {
            var hasClassName = new RegExp("(?:^|\s)" + className + "(?:0\s)");
            var allElements = document.getElementsByTagName("*");
            var results = [];
            
            var element;
            for (var i = 0; (element = allElements[i]) != null; i++) 
            {
                var elementClass = element.className;
                if (elementClass && elementClass.indexOf(className) != -1 && 
                    hasClassName.test(elementClass))
                {
                    results.push(element);
                }
            }
            
            return results;
        }
    }
}

-->
</script>

<!-- Ajax for Predict button -->
<script type='text/javascript' src='/saap/dap//predictAjax.js'></script>

<link rel='stylesheet' href='/css/font-awesome/css/font-awesome.min.css' />

<style type='text/css'>
<!--
p, td, th, li {  font-family: Helvetica,Arial; }

h1 
{  text-align: center;
   font: 24pt Helvetica,Arial
}

h2 { font: 18pt Helvetica,Arial;      }
h3 { font: bold 14pt Helvetica,Arial; }
h4 { font: 12pt Helvetica,Arial;      }
a  { outline: none;                   }

a span, a:active, a:visited, a:hover 
{  border: none; 
   color: #000000;
   text-decoration: none;
}

.mutation 
{  background: #666666;
   margin: 0em 0em 1em 0em;
   padding: 2pt;
}

.mutation td 
{  color: #FFFFFF;
   font: 12pt Helvetica, Arial;
}

.summary .ok      {background: #cccccc;}
.summary .bad     {background: #FF0000;}
.summary .unknown {background: #AAAAAA;}

.summary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: center;
}

.summary td, .summary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 10pt Helvetica,Arial;
}

.summary th { font: 12pt Helvetica,Arial; }

.summary th a:link, .summary th a:visited, .summary th a:hover, .summary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.mutantsummary .ok      {background: #00FF00;}
.mutantsummary .bad     {background: #FF0000;}
.mutantsummary .unknown {background: #AAAAAA;}

.mutantsummary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: left;
}

.mutantsummary td, .mutantsummary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 12pt Helvetica,Arial;
}

.mutantsummary th a:link, .mutantsummary th a:visited, .mutantsummary th a:hover, .mutantsummary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.footnote 
{  font: italic 8pt Helvetica,Arial;
}

.button 
{  font: bold 12pt Helvetica,Arial;
   background: #7A96BE;
   margin: 4pt 0pt;
   padding: 1pt 4pt 4pt 4pt;
   border: outset 2pt #000000;
}

.button a:link, .button a:visited, .button a:hover, .button a:active  
{  font: bold 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.analysis 
{  border: 1pt solid #000000;
   margin: 1em 0 0 0;
   padding: 0 0 0 0;
   background: #CCCCCC;
}

.analysis h3 
{  margin: 0 0 0 0;
   padding: 2pt;
   background: #666666;
   color: #FFFFFF;
}

.analysis h4 
{  margin: 0 0 0 0;
   padding: 0 2pt 2pt 2pt;
   background: #aaaaaa;
   color: #000000;
   border: solid 1pt #777777;
}

.analysis .left  { width: 49%; }
.analysis .right { width: 49%; }

.analysis .bad 
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #ff0000;
}

.analysis .ok  
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #eeeeee;
}

.analysis .unknown  
{  margin: 0;
   padding: 2pt;
   background: #AAAAAA;
   border-bottom: solid 1pt #7A96BE;
   border-left: solid 1pt #7A96BE;
   border-right: solid 1pt #7A96BE;
}

.analysis p 
{  margin: 0;
   padding: 0;
}

.analysis table { width: 100%; }
.analysis td    {vertical-align: top; }

.resultsFrame
{  background: #FFFFFF;
   border: solid 1px #666666;
   padding: 1em;
}

-->
</style>
</head>
<body>
<!-- <body onload='fixIE();'> -->
<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>

<h1>SAAP Analysis</h1>

<!-- END OF COMMON HEADER -->


<!-- INFO ON THE MUTATION -->
<div class='mutation'>
<table>
<tr><td>UniProt Entry:</td><td>P12883</td></tr>
<tr><td>Mutation:</td><td>R 1606-&gt;H</td></tr>
</table>
</div>
<!-- END OF INFO ON THE MUTATION -->

<!-- BEGIN SUMMARY -->
<h2>Summary</h2>

<div class='summary'>
<table>
    <tr>
        <th>PDB</th>
        <th>chain</th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation disrupts hydrogen bonding.');" 
        onmouseout="return nd();">HBonds</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was involved in a specific HBond or packing interaction with another protein chain or ligand.');" 
        onmouseout="return nd();">Binding</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a charged residue into the protein core.');" 
        onmouseout="return nd();">BuriedCharge</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a void in the protein core.');" 
        onmouseout="return nd();">Voids</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is annotated as a feature in UniProtKB/SwissProt.');" 
        onmouseout="return nd();">SProtFT</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a hydrophobic amino acid on the surface.');" 
        onmouseout="return nd();">SurfacePhobic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was in an interface.');" 
        onmouseout="return nd();">Interface</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a glycine which was adopting backbone torsions not allowed for other amino acids.');" 
        onmouseout="return nd();">Glycine</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces an amino acid which clashes with its surroundings.');" 
        onmouseout="return nd();">Clash</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a cis-proline.');" 
        onmouseout="return nd();">CisPro</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a proline when native residue was in a conformation not accessible to proline.');" 
        onmouseout="return nd();">Proline</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a hydrophilic residue into the protein core.');" 
        onmouseout="return nd();">CorePhilic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is significantly conserved.');" 
        onmouseout="return nd();">Impact</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was a cysteine involved in a disulphide bond.');" 
        onmouseout="return nd();">SSGeom</a>
        </th>
    </tr>
        <tr><td align='left'><a href='#P5chxA'>5chx</a> </td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5chxB'>5chx</a> </td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5cj4A'>5cj4</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5cj4B'>5cj4</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5cj4C'>5cj4</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>C</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5cj4D'>5cj4</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>D</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
</table>
<p class='footnote'>Hover over the column titles for an explanation.</p>
</div>

<!-- END SUMMARY -->
<!-- BEGIN STRUCTURAL ANALYSIS HEADER -->
<h2>Structural analysis</h2>

<div style='text-align: center;'>
<button onclick='expand_all();'>Expand all</button>
&nbsp;&nbsp; 
<button onclick='collapse_all();'>Collapse all</button>
</div>

<!-- END STRUCTURAL ANALYSIS HEADER -->

<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5chxA'>5chx Residue A1606 | Structure type: crystal | Resolution: 2.30A | R-factor: 21.00%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5chxAHBonds','P5chxAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5chxABinding','P5chxABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5chxABuriedCharge','P5chxABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 66.707%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5chxAVoids','P5chxAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 139.00</b>.<br /> 
The largest void in the native was 139.00.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5chxASProtFT','P5chxASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5chxASurfacePhobic','P5chxASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.4</b><br />
                 Relative accessibility of native residue: 66.707%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5chxAInterface','P5chxAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 66.707% 
    while the individual chain had an accessibility of 66.707%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5chxAGlycine','P5chxAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxAGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5chxAClash','P5chxAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -8.34&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5chxACisPro','P5chxACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5chxAProline','P5chxAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxAProline' class='collapsable' style='display: none'>
           The mutant residue was a HIS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5chxACorePhilic','P5chxACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.4<br />
                  Relative accessibility of native residue: 66.707%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5chxAImpact','P5chxAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5chxAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5chxASSGeom','P5chxASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5chxASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5chxB'>5chx Residue B1606 | Structure type: crystal | Resolution: 2.30A | R-factor: 21.00%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5chxBHBonds','P5chxBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5chxBBinding','P5chxBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5chxBBuriedCharge','P5chxBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 66.505%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5chxBVoids','P5chxBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 120.76</b>.<br /> 
The largest void in the native was 120.76.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5chxBSProtFT','P5chxBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5chxBSurfacePhobic','P5chxBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.4</b><br />
                 Relative accessibility of native residue: 66.505%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5chxBInterface','P5chxBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 66.505% 
    while the individual chain had an accessibility of 66.505%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5chxBGlycine','P5chxBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5chxBClash','P5chxBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -8.35&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5chxBCisPro','P5chxBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5chxBProline','P5chxBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBProline' class='collapsable' style='display: none'>
           The mutant residue was a HIS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5chxBCorePhilic','P5chxBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5chxBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.4<br />
                  Relative accessibility of native residue: 66.505%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5chxBImpact','P5chxBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5chxBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5chxBSSGeom','P5chxBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5chxBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5chxBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5cj4A'>5cj4 Residue A1606 | Structure type: crystal | Resolution: 3.10A | R-factor: 24.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5cj4AHBonds','P5cj4AHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4AHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4AHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5cj4ABinding','P5cj4ABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4ABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4ABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5cj4ABuriedCharge','P5cj4ABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4ABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4ABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 61.663%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5cj4AVoids','P5cj4AVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4AVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4AVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 134.57</b>.<br /> 
The largest void in the native was 134.57.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5cj4ASProtFT','P5cj4ASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4ASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4ASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5cj4ASurfacePhobic','P5cj4ASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4ASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4ASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.4</b><br />
                 Relative accessibility of native residue: 61.663%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5cj4AInterface','P5cj4AInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4AInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4AInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 61.663% 
    while the individual chain had an accessibility of 61.663%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5cj4AGlycine','P5cj4AGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4AGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4AGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5cj4AClash','P5cj4AClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4AClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4AClash' class='collapsable' style='display: none'>
           <b>The clash energy was -6.12&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5cj4ACisPro','P5cj4ACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4ACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4ACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5cj4AProline','P5cj4AProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4AProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4AProline' class='collapsable' style='display: none'>
           The mutant residue was a HIS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5cj4ACorePhilic','P5cj4ACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4ACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4ACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.4<br />
                  Relative accessibility of native residue: 61.663%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5cj4AImpact','P5cj4AImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4AImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5cj4AImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5cj4ASSGeom','P5cj4ASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4ASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5cj4ASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5cj4B'>5cj4 Residue B1606 | Structure type: crystal | Resolution: 3.10A | R-factor: 24.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5cj4BHBonds','P5cj4BHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5cj4BBinding','P5cj4BBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5cj4BBuriedCharge','P5cj4BBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 76.142%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5cj4BVoids','P5cj4BVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 218.68</b>.<br /> 
The largest void in the native was 292.22.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5cj4BSProtFT','P5cj4BSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5cj4BSurfacePhobic','P5cj4BSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.4</b><br />
                 Relative accessibility of native residue: 76.142%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5cj4BInterface','P5cj4BInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 76.142% 
    while the individual chain had an accessibility of 76.142%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5cj4BGlycine','P5cj4BGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5cj4BClash','P5cj4BClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BClash' class='collapsable' style='display: none'>
           <b>The clash energy was -3.29&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5cj4BCisPro','P5cj4BCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5cj4BProline','P5cj4BProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BProline' class='collapsable' style='display: none'>
           The mutant residue was a HIS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5cj4BCorePhilic','P5cj4BCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4BCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.4<br />
                  Relative accessibility of native residue: 76.142%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5cj4BImpact','P5cj4BImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5cj4BImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5cj4BSSGeom','P5cj4BSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4BSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5cj4BSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5cj4C'>5cj4 Residue C1606 | Structure type: crystal | Resolution: 3.10A | R-factor: 24.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5cj4CHBonds','P5cj4CHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5cj4CBinding','P5cj4CBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5cj4CBuriedCharge','P5cj4CBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 68.392%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5cj4CVoids','P5cj4CVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 135.76</b>.<br /> 
The largest void in the native was 135.76.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5cj4CSProtFT','P5cj4CSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5cj4CSurfacePhobic','P5cj4CSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.4</b><br />
                 Relative accessibility of native residue: 68.392%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5cj4CInterface','P5cj4CInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 68.392% 
    while the individual chain had an accessibility of 68.392%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5cj4CGlycine','P5cj4CGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5cj4CClash','P5cj4CClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CClash' class='collapsable' style='display: none'>
           <b>The clash energy was -5.37&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5cj4CCisPro','P5cj4CCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5cj4CProline','P5cj4CProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CProline' class='collapsable' style='display: none'>
           The mutant residue was a HIS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5cj4CCorePhilic','P5cj4CCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4CCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.4<br />
                  Relative accessibility of native residue: 68.392%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5cj4CImpact','P5cj4CImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5cj4CImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5cj4CSSGeom','P5cj4CSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4CSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5cj4CSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5cj4D'>5cj4 Residue D1606 | Structure type: crystal | Resolution: 3.10A | R-factor: 24.50% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5cj4DHBonds','P5cj4DHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5cj4DBinding','P5cj4DBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5cj4DBuriedCharge','P5cj4DBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 1</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 78.174%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5cj4DVoids','P5cj4DVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 164.54</b>.<br /> 
The largest void in the native was 284.79.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5cj4DSProtFT','P5cj4DSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5cj4DSurfacePhobic','P5cj4DSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                 <b>Mutant residue hydrophobicity: -0.4</b><br />
                 Relative accessibility of native residue: 78.174%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5cj4DInterface','P5cj4DInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 78.174% 
    while the individual chain had an accessibility of 78.174%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5cj4DGlycine','P5cj4DGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DGlycine' class='collapsable' style='display: none'>
           The native residue was a ARG not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5cj4DClash','P5cj4DClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DClash' class='collapsable' style='display: none'>
           <b>The clash energy was -3.82&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5cj4DCisPro','P5cj4DCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5cj4DProline','P5cj4DProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DProline' class='collapsable' style='display: none'>
           The mutant residue was a HIS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5cj4DCorePhilic','P5cj4DCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5cj4DCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: -1.8<br />
                  Mutant residue hydrophobicity: -0.4<br />
                  Relative accessibility of native residue: 78.174%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5cj4DImpact','P5cj4DImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5cj4DImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5cj4DSSGeom','P5cj4DSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5cj4DSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5cj4DSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<p>
<a href='./printpdf.cgi'>[Download Summary PDF]</a>
<a href='./printpdf.cgi?expand=1'>[Download Expanded PDF]</a>
</p>

<!-- COMMON FOOTER -->

</body>
</html>

