<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" 
   "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">

<head>
<title>SAAP analysis</title>

<script type='text/javascript' src='/js/getElementsByClassName.js'>
<!-- http://robertnyman.com -->
</script>
<script type="text/javascript" src="/js/overlib/overlib.js">
<!-- overLIB (c) Erik Bosrup -->
</script>

<script type='text/javascript'>
<!--


function toggle_text(obj) 
{
   var text = document.getElementById(obj);
   if ( text.style.display == 'inline' ) { text.style.display = 'none'; }
   else { text.style.display = 'inline'; }
}
function toggle_text_arrow(obj1, obj2) 
{
   var text = document.getElementById(obj1);
   var arrow = document.getElementById(obj2);
   if (( text.style.display == 'inline' ) || ( text.style.display == 'block' ))
   { 
      text.style.display = 'none'; 
      arrow.className = 'fliparrow fa fa-toggle-down';
   }
   else 
   { 
      text.style.display = 'inline'; 
      arrow.className = 'fliparrow fa fa-toggle-up';
   }
}
function expand_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'block';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-up';
   }
}

function collapse_all()
{
   var tags = document.getElementsByClassName('collapsable');
   for(var tag=0; tag<tags.length; tag++)
   {
      tags[tag].style.display = 'none';
   }

   var arrows = document.getElementsByClassName('fliparrow');
   for(var tag=0; tag<arrows.length; tag++)
   {
      arrows[tag].className = 'fliparrow fa fa-toggle-down';
   }
}

function fixIE()
{
//    document.getElementsByClassName("submit").disabled = false;

    if (document.getElementsByClassName == undefined) 
    {
        document.getElementsByClassName = function(className)
        {
            var hasClassName = new RegExp("(?:^|\s)" + className + "(?:0\s)");
            var allElements = document.getElementsByTagName("*");
            var results = [];
            
            var element;
            for (var i = 0; (element = allElements[i]) != null; i++) 
            {
                var elementClass = element.className;
                if (elementClass && elementClass.indexOf(className) != -1 && 
                    hasClassName.test(elementClass))
                {
                    results.push(element);
                }
            }
            
            return results;
        }
    }
}

-->
</script>

<!-- Ajax for Predict button -->
<script type='text/javascript' src='/saap/dap//predictAjax.js'></script>

<link rel='stylesheet' href='/css/font-awesome/css/font-awesome.min.css' />

<style type='text/css'>
<!--
p, td, th, li {  font-family: Helvetica,Arial; }

h1 
{  text-align: center;
   font: 24pt Helvetica,Arial
}

h2 { font: 18pt Helvetica,Arial;      }
h3 { font: bold 14pt Helvetica,Arial; }
h4 { font: 12pt Helvetica,Arial;      }
a  { outline: none;                   }

a span, a:active, a:visited, a:hover 
{  border: none; 
   color: #000000;
   text-decoration: none;
}

.mutation 
{  background: #666666;
   margin: 0em 0em 1em 0em;
   padding: 2pt;
}

.mutation td 
{  color: #FFFFFF;
   font: 12pt Helvetica, Arial;
}

.summary .ok      {background: #cccccc;}
.summary .bad     {background: #FF0000;}
.summary .unknown {background: #AAAAAA;}

.summary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: center;
}

.summary td, .summary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 10pt Helvetica,Arial;
}

.summary th { font: 12pt Helvetica,Arial; }

.summary th a:link, .summary th a:visited, .summary th a:hover, .summary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.mutantsummary .ok      {background: #00FF00;}
.mutantsummary .bad     {background: #FF0000;}
.mutantsummary .unknown {background: #AAAAAA;}

.mutantsummary table    
{  border-collapse: collapse;
   border: solid 1pt #000000;
   text-align: left;
}

.mutantsummary td, .mutantsummary th 
{  border: solid 1pt #000000;
   padding: 2pt;
   font: 12pt Helvetica,Arial;
}

.mutantsummary th a:link, .mutantsummary th a:visited, .mutantsummary th a:hover, .mutantsummary th a:active  
{  font: 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.footnote 
{  font: italic 8pt Helvetica,Arial;
}

.button 
{  font: bold 12pt Helvetica,Arial;
   background: #7A96BE;
   margin: 4pt 0pt;
   padding: 1pt 4pt 4pt 4pt;
   border: outset 2pt #000000;
}

.button a:link, .button a:visited, .button a:hover, .button a:active  
{  font: bold 12pt Helvetica,Arial;
   color: #000000;
   text-decoration: none;
}

.analysis 
{  border: 1pt solid #000000;
   margin: 1em 0 0 0;
   padding: 0 0 0 0;
   background: #CCCCCC;
}

.analysis h3 
{  margin: 0 0 0 0;
   padding: 2pt;
   background: #666666;
   color: #FFFFFF;
}

.analysis h4 
{  margin: 0 0 0 0;
   padding: 0 2pt 2pt 2pt;
   background: #aaaaaa;
   color: #000000;
   border: solid 1pt #777777;
}

.analysis .left  { width: 49%; }
.analysis .right { width: 49%; }

.analysis .bad 
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #ff0000;
}

.analysis .ok  
{  margin: 0;
   padding: 2pt;
   background: #eeeeee;
   border: solid 3px #eeeeee;
}

.analysis .unknown  
{  margin: 0;
   padding: 2pt;
   background: #AAAAAA;
   border-bottom: solid 1pt #7A96BE;
   border-left: solid 1pt #7A96BE;
   border-right: solid 1pt #7A96BE;
}

.analysis p 
{  margin: 0;
   padding: 0;
}

.analysis table { width: 100%; }
.analysis td    {vertical-align: top; }

.resultsFrame
{  background: #FFFFFF;
   border: solid 1px #666666;
   padding: 1em;
}

-->
</style>
</head>
<body>
<!-- <body onload='fixIE();'> -->
<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>

<h1>SAAP Analysis</h1>

<!-- END OF COMMON HEADER -->


<!-- INFO ON THE MUTATION -->
<div class='mutation'>
<table>
<tr><td>UniProt Entry:</td><td>P12883</td></tr>
<tr><td>Mutation:</td><td>M 1764-&gt;K</td></tr>
</table>
</div>
<!-- END OF INFO ON THE MUTATION -->

<!-- BEGIN SUMMARY -->
<h2>Summary</h2>

<div class='summary'>
<table>
    <tr>
        <th>PDB</th>
        <th>chain</th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation disrupts hydrogen bonding.');" 
        onmouseout="return nd();">HBonds</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was involved in a specific HBond or packing interaction with another protein chain or ligand.');" 
        onmouseout="return nd();">Binding</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a charged residue into the protein core.');" 
        onmouseout="return nd();">BuriedCharge</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a void in the protein core.');" 
        onmouseout="return nd();">Voids</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is annotated as a feature in UniProtKB/SwissProt.');" 
        onmouseout="return nd();">SProtFT</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a hydrophobic amino acid on the surface.');" 
        onmouseout="return nd();">SurfacePhobic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was in an interface.');" 
        onmouseout="return nd();">Interface</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a glycine which was adopting backbone torsions not allowed for other amino acids.');" 
        onmouseout="return nd();">Glycine</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces an amino acid which clashes with its surroundings.');" 
        onmouseout="return nd();">Clash</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native was a cis-proline.');" 
        onmouseout="return nd();">CisPro</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduces a proline when native residue was in a conformation not accessible to proline.');" 
        onmouseout="return nd();">Proline</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Mutation introduced a hydrophilic residue into the protein core.');" 
        onmouseout="return nd();">CorePhilic</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('The site of the mutation is significantly conserved.');" 
        onmouseout="return nd();">Impact</a>
        </th>
        <th><a href="javascript:void(0);" 
        onmouseover="return overlib('Native residue was a cysteine involved in a disulphide bond.');" 
        onmouseout="return nd();">SSGeom</a>
        </th>
    </tr>
        <tr><td align='left'><a href='#P5wj7A'>5wj7</a> </td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wj7B'>5wj7</a> </td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wjbA'>5wjb</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wjbB'>5wjb</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wjbC'>5wjb</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>C</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wjbD'>5wjb</a>  <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></td>
            <td>D</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wmeA'>5wme</a> </td>
            <td>A</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wmeB'>5wme</a> </td>
            <td>B</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wmeC'>5wme</a> </td>
            <td>C</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
        <tr><td align='left'><a href='#P5wmeD'>5wme</a> </td>
            <td>D</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='ok'>&nbsp;</td>
<td class='bad'>X</td>
<td class='ok'>&nbsp;</td>
        </tr>
</table>
<p class='footnote'>Hover over the column titles for an explanation.</p>
</div>

<!-- END SUMMARY -->
<!-- BEGIN STRUCTURAL ANALYSIS HEADER -->
<h2>Structural analysis</h2>

<div style='text-align: center;'>
<button onclick='expand_all();'>Expand all</button>
&nbsp;&nbsp; 
<button onclick='collapse_all();'>Collapse all</button>
</div>

<!-- END STRUCTURAL ANALYSIS HEADER -->

<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wj7A'>5wj7 Residue A1764 | Structure type: crystal | Resolution: 2.50A | R-factor: 20.60%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wj7AHBonds','P5wj7AHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7AHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7AHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wj7ABinding','P5wj7ABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7ABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7ABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wj7ABuriedCharge','P5wj7ABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7ABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7ABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 54.284%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wj7AVoids','P5wj7AVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7AVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7AVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 158.11</b>.<br /> 
The largest void in the native was 158.11.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wj7ASProtFT','P5wj7ASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7ASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7ASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wj7ASurfacePhobic','P5wj7ASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7ASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7ASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 54.284%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wj7AInterface','P5wj7AInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7AInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7AInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 54.284% 
    while the individual chain had an accessibility of 54.284%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wj7AGlycine','P5wj7AGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7AGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7AGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wj7AClash','P5wj7AClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7AClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7AClash' class='collapsable' style='display: none'>
           <b>The clash energy was -5.73&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wj7ACisPro','P5wj7ACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7ACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7ACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wj7AProline','P5wj7AProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7AProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7AProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wj7ACorePhilic','P5wj7ACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7ACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7ACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 54.284%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wj7AImpact','P5wj7AImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7AImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wj7AImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wj7ASSGeom','P5wj7ASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7ASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wj7ASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wj7B'>5wj7 Residue B1764 | Structure type: crystal | Resolution: 2.50A | R-factor: 20.60%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wj7BHBonds','P5wj7BHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wj7BBinding','P5wj7BBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wj7BBuriedCharge','P5wj7BBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 59.560%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wj7BVoids','P5wj7BVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 153.79</b>.<br /> 
The largest void in the native was 153.79.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wj7BSProtFT','P5wj7BSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wj7BSurfacePhobic','P5wj7BSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 59.560%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wj7BInterface','P5wj7BInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 59.560% 
    while the individual chain had an accessibility of 59.560%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wj7BGlycine','P5wj7BGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wj7BClash','P5wj7BClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BClash' class='collapsable' style='display: none'>
           <b>The clash energy was -5.18&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wj7BCisPro','P5wj7BCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wj7BProline','P5wj7BProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wj7BCorePhilic','P5wj7BCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wj7BCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 59.560%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wj7BImpact','P5wj7BImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wj7BImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wj7BSSGeom','P5wj7BSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wj7BSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wj7BSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wjbA'>5wjb Residue A1764 | Structure type: crystal | Resolution: 2.91A | R-factor: 25.10% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wjbAHBonds','P5wjbAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wjbABinding','P5wjbABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wjbABuriedCharge','P5wjbABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 65.114%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wjbAVoids','P5wjbAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 34.08</b>.<br /> 
The largest void in the native was 34.08.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wjbASProtFT','P5wjbASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wjbASurfacePhobic','P5wjbASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 65.114%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wjbAInterface','P5wjbAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 65.114% 
    while the individual chain had an accessibility of 65.114%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wjbAGlycine','P5wjbAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbAGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wjbAClash','P5wjbAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -8.03&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wjbACisPro','P5wjbACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wjbAProline','P5wjbAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbAProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wjbACorePhilic','P5wjbACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 65.114%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wjbAImpact','P5wjbAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wjbAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wjbASSGeom','P5wjbASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wjbASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wjbB'>5wjb Residue B1764 | Structure type: crystal | Resolution: 2.91A | R-factor: 25.10% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wjbBHBonds','P5wjbBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wjbBBinding','P5wjbBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wjbBBuriedCharge','P5wjbBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 68.594%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wjbBVoids','P5wjbBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 36.84</b>.<br /> 
The largest void in the native was 36.84.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wjbBSProtFT','P5wjbBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wjbBSurfacePhobic','P5wjbBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 68.594%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wjbBInterface','P5wjbBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 68.594% 
    while the individual chain had an accessibility of 68.594%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wjbBGlycine','P5wjbBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wjbBClash','P5wjbBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -1.06&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wjbBCisPro','P5wjbBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wjbBProline','P5wjbBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wjbBCorePhilic','P5wjbBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 68.594%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wjbBImpact','P5wjbBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wjbBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wjbBSSGeom','P5wjbBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wjbBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wjbC'>5wjb Residue C1764 | Structure type: crystal | Resolution: 2.91A | R-factor: 25.10% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wjbCHBonds','P5wjbCHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wjbCBinding','P5wjbCBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wjbCBuriedCharge','P5wjbCBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 68.135%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wjbCVoids','P5wjbCVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 23.86</b>.<br /> 
The largest void in the native was 23.86.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wjbCSProtFT','P5wjbCSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wjbCSurfacePhobic','P5wjbCSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 68.135%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wjbCInterface','P5wjbCInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 68.135% 
    while the individual chain had an accessibility of 68.135%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wjbCGlycine','P5wjbCGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wjbCClash','P5wjbCClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCClash' class='collapsable' style='display: none'>
           <b>The clash energy was -1.99&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wjbCCisPro','P5wjbCCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wjbCProline','P5wjbCProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wjbCCorePhilic','P5wjbCCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbCCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 68.135%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wjbCImpact','P5wjbCImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wjbCImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wjbCSSGeom','P5wjbCSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbCSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wjbCSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wjbD'>5wjb Residue D1764 | Structure type: crystal | Resolution: 2.91A | R-factor: 25.10% <a href="javascript:void(0);" 
        onmouseover="return overlib('The structure has poor resolution. Detailed structural analyses should be interpreted with caution.');" 
        onmouseout="return nd();"><span style='color: orange;' class='fa fa-warning'></span></a></h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wjbDHBonds','P5wjbDHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wjbDBinding','P5wjbDBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wjbDBuriedCharge','P5wjbDBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 65.997%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wjbDVoids','P5wjbDVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 49.05</b>.<br /> 
The largest void in the native was 49.05.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wjbDSProtFT','P5wjbDSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wjbDSurfacePhobic','P5wjbDSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 65.997%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wjbDInterface','P5wjbDInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 65.997% 
    while the individual chain had an accessibility of 65.997%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wjbDGlycine','P5wjbDGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wjbDClash','P5wjbDClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDClash' class='collapsable' style='display: none'>
           <b>The clash energy was -6.35&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wjbDCisPro','P5wjbDCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wjbDProline','P5wjbDProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wjbDCorePhilic','P5wjbDCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wjbDCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 65.997%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wjbDImpact','P5wjbDImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wjbDImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wjbDSSGeom','P5wjbDSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wjbDSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wjbDSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wmeA'>5wme Residue A1764 | Structure type: crystal | Resolution: 2.30A | R-factor: 22.60%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wmeAHBonds','P5wmeAHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeAHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeAHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wmeABinding','P5wmeABindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeABindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeABinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wmeABuriedCharge','P5wmeABuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeABuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeABuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 46.076%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wmeAVoids','P5wmeAVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeAVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeAVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 23.04</b>.<br /> 
The largest void in the native was 23.04.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wmeASProtFT','P5wmeASProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeASProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeASProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wmeASurfacePhobic','P5wmeASurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeASurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeASurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 46.076%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wmeAInterface','P5wmeAInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeAInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeAInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 46.076% 
    while the individual chain had an accessibility of 46.076%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wmeAGlycine','P5wmeAGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeAGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeAGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wmeAClash','P5wmeAClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeAClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeAClash' class='collapsable' style='display: none'>
           <b>The clash energy was -7.98&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wmeACisPro','P5wmeACisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeACisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeACisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wmeAProline','P5wmeAProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeAProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeAProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wmeACorePhilic','P5wmeACorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeACorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeACorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 46.076%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wmeAImpact','P5wmeAImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeAImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wmeAImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wmeASSGeom','P5wmeASSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeASSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wmeASSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wmeB'>5wme Residue B1764 | Structure type: crystal | Resolution: 2.30A | R-factor: 22.60%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wmeBHBonds','P5wmeBHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wmeBBinding','P5wmeBBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wmeBBuriedCharge','P5wmeBBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 65.902%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wmeBVoids','P5wmeBVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 31.56</b>.<br /> 
The largest void in the native was 31.56.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wmeBSProtFT','P5wmeBSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wmeBSurfacePhobic','P5wmeBSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 65.902%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wmeBInterface','P5wmeBInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 65.902% 
    while the individual chain had an accessibility of 65.902%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wmeBGlycine','P5wmeBGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wmeBClash','P5wmeBClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBClash' class='collapsable' style='display: none'>
           <b>The clash energy was -4.36&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wmeBCisPro','P5wmeBCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wmeBProline','P5wmeBProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wmeBCorePhilic','P5wmeBCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeBCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 65.902%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wmeBImpact','P5wmeBImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wmeBImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wmeBSSGeom','P5wmeBSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeBSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wmeBSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wmeC'>5wme Residue C1764 | Structure type: crystal | Resolution: 2.30A | R-factor: 22.60%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wmeCHBonds','P5wmeCHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wmeCBinding','P5wmeCBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wmeCBuriedCharge','P5wmeCBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 58.966%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wmeCVoids','P5wmeCVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 32.78</b>.<br /> 
The largest void in the native was 32.78.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wmeCSProtFT','P5wmeCSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wmeCSurfacePhobic','P5wmeCSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 58.966%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wmeCInterface','P5wmeCInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 58.966% 
    while the individual chain had an accessibility of 58.966%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wmeCGlycine','P5wmeCGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wmeCClash','P5wmeCClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCClash' class='collapsable' style='display: none'>
           <b>The clash energy was -2.67&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wmeCCisPro','P5wmeCCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wmeCProline','P5wmeCProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wmeCCorePhilic','P5wmeCCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeCCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 58.966%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wmeCImpact','P5wmeCImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wmeCImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wmeCSSGeom','P5wmeCSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeCSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wmeCSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<!-- BEGIN A STRUCTURAL ANALYSIS -->
<div class='analysis'>
<h3 id='P5wmeD'>5wme Residue D1764 | Structure type: crystal | Resolution: 2.30A | R-factor: 22.60%</h3>
<table>
<colgroup>
<col class='left' />
<col class='right' />
</colgroup>
<tr>
    <td>
        <h4>HBonds
        <a href="javascript:toggle_text_arrow('P5wmeDHBonds','P5wmeDHBondsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDHBondsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDHBonds' class='collapsable' style='display: none'>
           The native residue was not involved in a hydrogen bond
        </p>
        </div>
    </td>
    <td>
        <h4>Binding
        <a href="javascript:toggle_text_arrow('P5wmeDBinding','P5wmeDBindingArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDBindingArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDBinding' class='collapsable' style='display: none'>
           The residue was not involved in a specific binding interaction
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>BuriedCharge
        <a href="javascript:toggle_text_arrow('P5wmeDBuriedCharge','P5wmeDBuriedChargeArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDBuriedChargeArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDBuriedCharge' class='collapsable' style='display: none'>
           <b>Native residue charge: 0</b><br />
               <b>Mutant residue charge: 1</b><br />
               Relative accessibility of native residue: 66.924%
        </p>
        </div>
    </td>
    <td>
        <h4>Voids
        <a href="javascript:toggle_text_arrow('P5wmeDVoids','P5wmeDVoidsArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDVoidsArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDVoids' class='collapsable' style='display: none'>
           After the mutation <b>the largest void was of size 45.45</b>.<br /> 
The largest void in the native was 45.45.<br /> 
90% of proteins have no voids &gt;275, but we do not consider a large 
void to be damaging if the native structure had large voids.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>SProtFT
        <a href="javascript:toggle_text_arrow('P5wmeDSProtFT','P5wmeDSProtFTArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDSProtFTArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDSProtFT' class='collapsable' style='display: none'>
           The residue was not annotated as a 'feature' in UniProtKB/SwissProt
        </p>
        </div>
    </td>
    <td>
        <h4>SurfacePhobic
        <a href="javascript:toggle_text_arrow('P5wmeDSurfacePhobic','P5wmeDSurfacePhobicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDSurfacePhobicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDSurfacePhobic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                 <b>Mutant residue hydrophobicity: -1.1</b><br />
                 Relative accessibility of native residue: 66.924%<br /><br />
                 Hydrophobicity values &lt;0 are hydrophilic and a threshold of 20% relative accessibility is used to define a residue as on the surface.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Interface
        <a href="javascript:toggle_text_arrow('P5wmeDInterface','P5wmeDInterfaceArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDInterfaceArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDInterface' class='collapsable' style='display: none'>
           Interfaces are defined by a difference in solvent
    accessibility between a complex and the individual chain in the crystal structure.<br />
    In the complex, this residue had a relative accessability of 66.924% 
    while the individual chain had an accessibility of 66.924%, <b>a
    difference of 0.000%</b>.<br />
    A difference of &gt;10% is taken as indicative of an interface residue.<br />
    Interfaces may be with another protein chain or a ligand.
        </p>
        </div>
    </td>
    <td>
        <h4>Glycine
        <a href="javascript:toggle_text_arrow('P5wmeDGlycine','P5wmeDGlycineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDGlycineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDGlycine' class='collapsable' style='display: none'>
           The native residue was a MET not a Glycine
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Clash
        <a href="javascript:toggle_text_arrow('P5wmeDClash','P5wmeDClashArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDClashArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDClash' class='collapsable' style='display: none'>
           <b>The clash energy was -7.67&nbsp;kcal/mol.</b><br /> 99% of sidechains 
in real proteins have an energy less than 34.33&nbsp;kcal/mol.<br />Consequently energies 
&gt;34.33&nbsp;kcal/mol and &lt;50&nbsp;kcal/mol can be considered mild clashes, 
50-100&nbsp;kcal/mol medium clashes, &gt;100&nbsp;kcal/mol severe clashes. Note 
that clash energies can be extremely high (&gt;&gt;100000&nbsp;kcal/mol)
        </p>
        </div>
    </td>
    <td>
        <h4>CisPro
        <a href="javascript:toggle_text_arrow('P5wmeDCisPro','P5wmeDCisProArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDCisProArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDCisPro' class='collapsable' style='display: none'>
           The native residue was not a cis-proline
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Proline
        <a href="javascript:toggle_text_arrow('P5wmeDProline','P5wmeDProlineArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDProlineArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDProline' class='collapsable' style='display: none'>
           The mutant residue was a LYS not a Proline
        </p>
        </div>
    </td>
    <td>
        <h4>CorePhilic
        <a href="javascript:toggle_text_arrow('P5wmeDCorePhilic','P5wmeDCorePhilicArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDCorePhilicArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified</p>
        <p id='P5wmeDCorePhilic' class='collapsable' style='display: none'>
           Native residue hydrophobicity: 0.26<br />
                  Mutant residue hydrophobicity: -1.1<br />
                  Relative accessibility of native residue: 66.924%<br /><br />
                  Hydrophobicity values &lt;0 are hydrophilic.
        </p>
        </div>
    </td>
</tr>
<tr>
    <td>
        <h4>Impact
        <a href="javascript:toggle_text_arrow('P5wmeDImpact','P5wmeDImpactArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDImpactArrow' ></span>
        </a> 
        </h4>
        <div class='bad'>
        <p>The mutation was at a site flagged as 'highly conserved'.</p>
        <p id='P5wmeDImpact' class='collapsable' style='display: none'>
           In an alignment of 9 functionally equivalent protein sequences,
                         <b>the conservation score at this position was 100%.</b><br />
                         Analysis of the sequences in this alignment defined a threshold of 79% as 
                         being significantly conserved.<br />Note that since there are &lt;10 sequences in the alignment, this result should be treated with caution!
        </p>
        </div>
    </td>
    <td>
        <h4>SSGeom
        <a href="javascript:toggle_text_arrow('P5wmeDSSGeom','P5wmeDSSGeomArrow')">
        <span class='fliparrow fa fa-toggle-down ' id='P5wmeDSSGeomArrow' ></span>
        </a> 
        </h4>
        <div class='ok'>
        <p>No problems identified.</p>
        <p id='P5wmeDSSGeom' class='collapsable' style='display: none'>
           The native residue was not a cysteine in a disulphide bond.
        </p>
        </div>
    </td>
</tr>
</table>
</div> <!-- analysis -->
<!-- END A STRUCTURAL ANALYSIS -->
<!-- ********************************************************************* -->
<p>
<a href='./printpdf.cgi'>[Download Summary PDF]</a>
<a href='./printpdf.cgi?expand=1'>[Download Expanded PDF]</a>
</p>

<!-- COMMON FOOTER -->

</body>
</html>

